src/demo-app/a11y/sidenav/dual-sidenav-a11y.html

```diff
@@ -0,0 +1,41 @@
+<mat-toolbar color="primary" role="header" aria-label="Dual sidenav app">
+  <button mat-icon-button aria-label="Navigation menu" (click)="startSnav.toggle()">
+    <mat-icon aria-hidden="true">menu</mat-icon>
+  </button>
+  <h1 class="a11y-demo-sidenav-app-name">Dual Sidenav App</h1>
+  <span class="a11y-demo-sidenav-spacer" aria-hidden="true"></span>
+  <button mat-icon-button aria-label="Playlist menu" (click)="endSnav.toggle()">
```

Comment:
Is there a way to focus on the playlist items when toggle sidenav? The focus stays in the toggle icon button, and user need to go through the left sidenav to reach the second sidenav.